### DISCUSSION TITLE
Thus... I have arrived

### DISCUSSION THREAD

**Discussion by DennyDaVjncj** [zediious/raptor-web#155] [Show and tell]
<img width="1435" alt="Screenshot 2023-12-29 at 22 02 33" src="https://github.com/zediious/raptor-web/assets/62162419/ba7ce241-4d4e-42e5-b4f8-04179cc67ea0">

**Comment by DennyDaVjncj** [comment]
Very excited for this project bro, very excited!

**Reply by zediious** [reply]
Awesome man! Definitely let me know if you have any questions. I have absolutely no contributor docs currently, I can definitely point you here, as you'll need it to build the virtual environment that's supplied. https://www.anaconda.com/download/

**Reply by DennyDaVjncj** [reply]
What's up @zediious, as you know by now I'm sure, they sacked me! But I also want to let you know that I'm involved in another independent engineering project, that has some pretty cool financial implications, so that's where my attention has gone! Eventually I will circle back to this effort once things die down with this project I'm on.

Here's my number: 4076137858 - I'd love to keep touch bruv!
